docs/conf.py

```diff
@@ -100,78 +103,81 @@
 import guzzle_sphinx_theme
 
 html_theme_path = guzzle_sphinx_theme.html_theme_path()
-html_theme = 'guzzle_sphinx_theme'
+html_theme = "guzzle_sphinx_theme"
 
 
 def set_rst_settings(app):
-    app.env.settings.update({
-        'field_name_limit': 0,
-        'option_limit': 0,
-    })
+    app.env.settings.update(
+        {
+            "field_name_limit": 0,
+            "option_limit": 0,
+        }
+    )
```

Comment:
There was a trailing comma after option_limit. This would have been compressed to one line:
```py3
app.env.settings.update({
    'field_name_limit': 0,
    'option_limit': 0
})
```

Reply:
I somehow dislike how black uses (absence of) trailing commas to trigger other ways of formatting.